api/dbadapters/bigquery.ts

```diff
@@ -247,7 +247,7 @@ export class BigQueryDbAdapter implements IDbAdapter {
       const metadata = tableMetadataMap.get(action.target);
       const persistTable = dataform.PersistedTableMetadata.create({
         target: action.target,
-        lastUpdatedMillis: Long.fromString(metadata.lastModifiedTime),
+        lastUpdatedMillis: Long.fromString(metadata?.lastModifiedTime || "0"),
```

Comment:
When I ran the tests for the first time they threw because `metadata` was null; I think this is the right fix for that.

Reply:
> I don't understand why your change would affect this... I'm definitely a bit worried about this change, can you clarify under what condition(s) it happens?

Adding a new integration test file, the test will throw the first time from this line, saying that metadata is null and lastModifiedTime can't be read. Not needed to make tests pass, just figured I'd fix; have removed though so it can be investigated closer.